server/platform/services/cache/lru.go

```diff
@@ -256,3 +256,10 @@ func (l *LRU) removeElement(e *list.Element) {
 	}
 	delete(l.items, kv.key)
 }
+
+func (*LRU) Increment(key string, val int) error {
+	panic("not implemented")
+}
+func (*LRU) Decrement(key string, val int) error {
+	panic("not implemented")
+}
```

Comment:
I'm a bit concerned about this :thinking: If these two functions are not common to both caches, I'd say they shouldn't be in the interface at all.

Reply:
The thing is, I didn't want to cast to (*Redis) and then call the functions. But maybe that's the right approach here?

Reply:
Yeah, that can potentially panic as well, but I feel that's slightly better. If one needs to panic on "not implemented" for some implementations of the interface, then the interface is too large.

Reply:
Actually, I have another idea. I can create a larger interface containing the Increment/Decrement method, and then check for that. Let me do that.

Reply:
@agarciamontoro - lmk how do the new changes look to you.

Reply:
Ah, that's better! Thanks :)